Client/Frontend/Browser/BrowserViewController/BrowserViewController+ReaderMode.swift

```diff
@@ -157,23 +156,18 @@ extension BrowserViewController {
         }
     }
 
-    @objc func dynamicFontChanged(_ notification: Notification) {
-        guard notification.name == .DynamicFontChanged else { return }
```

Comment:
If that's removed, do we still adjust when dynamic font type is changed? (was it even properly working before?)

Reply:
It was not working properly before, I try to fix the accessibility for the views but this work will increase the size of this ticket significantly so it was agreed to create a separate ticket to improve Accessibility in reader mode